table/table_builder.h

```diff
@@ -56,7 +56,8 @@ struct TableBuilderOptions {
       CompressionType _compression_type,
       const CompressionOptions& _compression_opts,
       const std::string* _compression_dict, bool _skip_filters,
-      const std::string& _column_family_name, int _level)
+      const std::string& _column_family_name, int _level,
+      const uint64_t _creation_time = 0)
```

Comment:
oh I just remembered an issue last time I did a feature check (for DeleteRange) based on the table format name. There were external customers with custom table formats who also wanted to support my feature, but couldn't since they used a different name. They contributed this fix: #2035 (I never pulled it due to missing CLA). It doesn't really matter to me which way you choose, just telling you my experience last time :).